src/components/ui.js

```diff
@@ -1,7 +1,7 @@
 /* globals AFRAME THREE */
 AFRAME.registerComponent('ui', {
   schema: { brightness: { default: 1.0, max: 1.0, min: 0.0 } },
-  dependencies: ['ui-raycaster'],
+  dependencies: ['ui-raycaster', 'screenshot-camera', 'brush'],
```

Comment:
'brush' is a dependency of `paint-controls`. Any reason do you need it here as a dependency? I would do a new component `screenshot-controls` that it's enabled when clicking on the `screenshot` button. The `ui` component manages what controls are enable/disable. For instance: You click on `screenshot` button then you disable `paint-controls` and enable `screenshot-controls`